packages/scripts/README.md

```diff
@@ -54,9 +58,13 @@ This is how you execute the script with presented setup:
 
 * `npm run build` - builds the code for production.
 
+#### Advanced information
+
+This script uses [webpack](https://webpack.js.org/) behind the scenes. It'll lookup for a webpack config in the top-level directory of your package and will use it if it finds one. If none is found, it'll use the default config bundled within `@wordpress/scripts` packages. Learn more in the [Advanced Usage](#advanced-usage) section.
```

Comment:
I'd either use _"look something up"_ or _"look for something"_. In this case, probably _"it'll look for a webpack config"_.

Minor: I suggest _"the default config provided by `@wordpress/scripts`"_ rather than _"bundled within"_.